src/js/controller/model.js

```diff
@@ -148,7 +148,13 @@ define([
                     this.setCurrentAudioTrack(data.currentTrack, data.tracks);
                     break;
                 case 'subtitlesTrackChanged':
-                    this.setVideoSubtitleTrack(data.currentTrack, data.tracks);
+                    var adState = this.get('adState');
+
+                    // We only want to update the tracks ands persist the select track
+                    // when an ad is not playing
+                    if (!_.isString(adState)) {
+                        this.persistVideoSubtitleTrack(data.currentTrack, data.tracks);
+                    }
```

Comment:
Hmm... so the ads provider is emitting this event? There shouldn't be any captions to persist. I question why this is necessary.